.github/workflows/test-suite-desktop-e2e.yml

```
@@ -36,20 +36,61 @@ jobs:
+            "https://api.github.com/repos/$REPO/actions/runs/$LATEST_SUCCESS_RUN_ID/jobs")
+
+          # Check if this specific test group passed
+          JOB_STATUS=$(echo "$PREVIOUS_JOBS" | jq -r --arg JOB_NAME "$JOB_NAME" '.jobs[] | select(.name==$JOB_NAME) | .conclusion')
+
+          if [[ "$JOB_STATUS" == "success" ]]; then
+            echo "Job '$JOB_NAME' already passed in run $LATEST_SUCCESS_RUN_ID. Skipping."
+            exit 0
+          else
+            echo "Job '$JOB_NAME' was not successful in previous runs. Running tests."
+          fi
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)